assets/src/edit-story/components/panels/background.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useStory from '../../app/story/useStory';
+import { ActionButton } from '../button';
+import { SimplePanel } from './panel';
+
+// @todo Only display if one element selected.
```

Comment:
This is easy to do right away, no? E.g. `if (selectedElements.length > 1) {return null?} `?

Reply:
Yes, it is. However, I was thinking that it would be good to instead have a general logic for the panels support, such as for multi-selection, which could then be used directly in `panels/index.js` instead of the panel's component having to worry about when to be displayed.

Will add a comment + condition to only display for single selection to that file instead.

Reply:
I see your point. And, yes, I think that would be a better set up.